src/modules/add-task.js

```diff
@@ -1 +1,134 @@
-// will be used for the next part of the project.
+const taskInput = document.querySelector('.addtolist');
+const taskBox = document.querySelector('.task-box');
+
+const ClearAll = document.querySelector('.clearCompleted');
+
+const completed = false;
+let index = 0;
+let list = [];
+
+function showTodo() {
+  let liItem = '';
+  list.forEach((element) => {
+    if (element.completed === true) {
+      liItem += `<li class="task">
+                                    <input type="checkbox" class="checkme" checked onchange ="CheckMe(${element.index});" id="check${element.index}">
+                    
+                                    <input type="text" value="${element.description}" class="tasks" id="item${element.index}" readonly>                                    <i class="fa-solid uil-ellipsis-v edit " id="edit${element.index}" onclick="editItem(${element.index});"></i>
+                                    <i class="fa-solid fa-trash delete" id="delete${element.index}" onclick="removeItem(${element.index});"></i>
+                                           
+                                        </li>`;
+    } else {
+      liItem += `<li class="task">
+
+                            <input type="checkbox" class="checkme" onclick="CheckMe(${element.index});" id="check${element.index}">
```

Comment:
- [ ] Kindly use onChange instead of onClick as required by the previous reviewer to make you project complete :+1: